### DISCUSSION TITLE
Select2_multiple field with same pivot table but different fields on the same form

### DISCUSSION THREAD

**Discussion by aminsadiq350** [Laravel-Backpack/community-forum#320] [General]
I-am in the process of making this project where I have two (or more) fields having the same pivot table and am using the relationship type on backpack. What i want to do is have these two fields work independently.

So let's say I want to have one field where the user selects thickness while in the other field the user can select shading suppose.

The issue is that these two fields have the same pivot table and the same relationship set up in the model and therefore only one of them would show up in the same form. I do know that HTML forms replace the fields with the same "name" attribute but I'd like to know how can i get around solving this issue and have the fields work independently maintaining the select2_multiple functionality.

Here's the link to my complete question which I posted on stackoverflow:
https://stackoverflow.com/questions/62728683/backpack-for-laravel-select2-multiple-field-with-same-pivot-table-but-different 

P.S: Need to get this project done ASAP please suggest a solution.

**Comment by jcastroa87** [comment]
Hi.

I will close this discussion, Tabacitu answer on Stackoverflow cover this question:
> You can read more about the Create & Update in the docs, specifically the "How it works" sections - [backpackforlaravel.com/docs/4.1/…](https://backpackforlaravel.com/docs/4.1/crud-operation-create#how-it-works) but basically you can create a method in your CrudController called store() or update() and overwrite the saving process. It's possible you don't need that at all though, if your input in the view contains the EXACT name as the column in the DB, and the content is EXCTLY as you want it stored in the DB, it'll just work. If you want to manipulate before storing, I recommend Laravel accessors and mutators, in the Model.


Cheers.
